src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const expensesService = require('../services/expenses.services');
+const userService = require('../services/users.services');
+const { expensesFilter } = require('../utils/expensesFilter');
+
+const getAll = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+    categories,
+  } = req.query;
+
+  const expenses = expensesFilter(userId, from, to, categories);
```

Comment:
Is there any point of destructuring this data here?

Reply:
Good point, you don't need destructuring